subcommand/create-federation-secret/command.go

```
@@ -295,7 +295,7 @@ func (c *Command) replicationToken(logger hclog.Logger) ([]byte, error) {
 	// requires all servers to be running and a leader elected.
 	// This will run forever but it's running as a Helm hook so Helm will timeout
 	// after a configurable time period.
-	backoff.Retry(func() error {
+	_ = backoff.Retry(func() error {
```

should we not ignore this error?